agent/structs/config_entry_routes.go

```diff
@@ -87,12 +88,22 @@ func (e *HTTPRouteConfigEntry) Normalize() error {
 		for j, match := range rule.Matches {
 			rule.Matches[j] = normalizeHTTPMatch(match)
 		}
+
+		for j, service := range rule.Services {
+			rule.Services[j] = normalizeHTTPService(service)
+		}
 		e.Rules[i] = rule
 	}
 
 	return nil
 }
 
+func normalizeHTTPService(service HTTPService) HTTPService {
+	service.EnterpriseMeta.Normalize()
+
+	return service
+}
```

Comment:
Any reason this follows a different pattern than everywhere else? Aside from that nit, LGTM assuming that `EnterpriseMeta.Normalize()` mutates the struct.

Reply:
It does mutate the struct -- verified it and double checked that normalize takes a pointer receiver 😅.

I just did this pattern since this is how the rest of the HTTPRoute normalization/validation code is done since there are substantially more subsections to cover than the other resources.